contracts/Optimism_SpokePool.sol

```diff
@@ -122,13 +137,17 @@ contract Optimism_SpokePool is CrossDomainEnabled, SpokePool {
             WETH9(relayerRefundLeaf.l2TokenAddress).withdraw(relayerRefundLeaf.amountToReturn); // Unwrap into ETH.
             relayerRefundLeaf.l2TokenAddress = l2Eth; // Set the l2TokenAddress to ETH.
         }
-        IL2ERC20Bridge(Lib_PredeployAddresses.L2_STANDARD_BRIDGE).withdrawTo(
-            relayerRefundLeaf.l2TokenAddress, // _l2Token. Address of the L2 token to bridge over.
-            hubPool, // _to. Withdraw, over the bridge, to the l1 pool contract.
-            relayerRefundLeaf.amountToReturn, // _amount.
-            l1Gas, // _l1Gas. Unused, but included for potential forward compatibility considerations
-            "" // _data. We don't need to send any data for the bridging action.
-        );
+        IL2ERC20Bridge(
+            tokenBridges[relayerRefundLeaf.l2TokenAddress] == address(0)
+                ? Lib_PredeployAddresses.L2_STANDARD_BRIDGE
+                : tokenBridges[relayerRefundLeaf.l2TokenAddress]
+        ).withdrawTo(
+                relayerRefundLeaf.l2TokenAddress, // _l2Token. Address of the L2 token to bridge over.
+                hubPool, // _to. Withdraw, over the bridge, to the l1 pool contract.
+                relayerRefundLeaf.amountToReturn, // _amount.
+                l1Gas, // _l1Gas. Unused, but included for potential forward compatibility considerations
+                "" // _data. We don't need to send any data for the bridging action.
```

Comment:
since this can be used in some "nonstandard" ERC20 withdrawal case (as referenced for DIA and SNX's custom withdrawals above) - does it make sense to actually leave this data parameter open for input? If it's a standard optimism withdrawal then `data` can just be ignored, but otherwise it could be useful if any nonstandard withdrawal needs more parameters than the ones included already here

Reply:
After talking this out on slack, we agreed it's best to leave as is, and update the contract with an upgrade if needed in the future